docs/run/run-primitives-batch.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Run primitives in a batch
+description: How to run primitives in batch mode. 
+
+---
+
+# Run primitives in a batch
+
+Batch mode is ideal for running experiments closely together to avoid device drifts, that is, to maintain device characterization.  All jobs must be provided at the outset. If you want to submit iterative job, use [sessions](sessions) instead.
```

Comment:
```suggestion
Batch mode is ideal for running experiments closely together to avoid device drifts, that is, to maintain device characterization.  All jobs must be provided at the outset. If you want to submit iterative jobs, use [sessions](sessions) instead.
```